_includes/pbc/all/install-features/202311.0/unified-commerce/install-the-warehouse-picking-feature.md

```diff
@@ -108,11 +134,54 @@ Make sure the following changes have been triggered in transfer objects:
+| OauthRequest.redirectUri               | property | created | src/Generated/Shared/Transfer/OauthRequestTransfer       |
+| OauthRequest.state                     | property | created | src/Generated/Shared/Transfer/OauthRequestTransfer       |
+| OauthRequest.code                      | property | created | src/Generated/Shared/Transfer/OauthRequestTransfer       |
+| OauthRequest.codeChallenge             | property | created | src/Generated/Shared/Transfer/AuthCodeAttributesTransfer |
+| OauthRequest.codeChallengeMethod       | property | created | src/Generated/Shared/Transfer/OauthRequestTransfer       |
+| OauthRequest.codeVerifier              | property | created | src/Generated/Shared/Transfer/OauthRequestTransfer       |
+| OauthResponse.state                    | property | created | src/Generated/Shared/Transfer/OauthResponseTransfer      |
+| OauthResponse.code                     | property | created | src/Generated/Shared/Transfer/OauthResponseTransfer      |
+| ApiTokenAttributes.code                | property | created | src/Generated/Shared/Transfer/ApiTokenAttributesTransfer |
+| ApiTokenAttributes.clientId            | property | created | src/Generated/Shared/Transfer/ApiTokenAttributesTransfer |
+| ApiTokenAttributes.redirectUri         | property | created | src/Generated/Shared/Transfer/ApiTokenAttributesTransfer |
+| ApiTokenAttributes.codeVerifier        | property | created | src/Generated/Shared/Transfer/ApiTokenAttributesTransfer |
+| AuthCodeAttributes.username            | property | created | src/Generated/Shared/Transfer/AuthCodeAttributesTransfer |
+| AuthCodeAttributes.password            | property | created | src/Generated/Shared/Transfer/AuthCodeAttributesTransfer |
+| AuthCodeAttributes.responseType        | property | created | src/Generated/Shared/Transfer/AuthCodeAttributesTransfer |
+| AuthCodeAttributes.clientId            | property | created | src/Generated/Shared/Transfer/AuthCodeAttributesTransfer |
+| AuthCodeAttributes.scope               | property | created | src/Generated/Shared/Transfer/AuthCodeAttributesTransfer |
+| AuthCodeAttributes.redirectUri         | property | created | src/Generated/Shared/Transfer/AuthCodeAttributesTransfer |
+| AuthCodeAttributes.state               | property | created | src/Generated/Shared/Transfer/AuthCodeAttributesTransfer |
+| AuthCodeAttributes.codeChallenge       | property | created | src/Generated/Shared/Transfer/AuthCodeAttributesTransfer |
+| AuthCodeAttributes.codeChallengeMethod | property | created | src/Generated/Shared/Transfer/AuthCodeAttributesTransfer |
+| AuthCode.identifier                    | property | created | src/Generated/Shared/Transfer/AuthCodeTransfer           |
+| AuthCode.expiresAt                     | property | created | src/Generated/Shared/Transfer/AuthCodeTransfer           |
+
 {% endinfo_block %}
 
+
 ## 3) Set up configuration
```

Comment:
I split this section into subsections. Please make sure the subsection titles are relevant